packages/@react-aria/selection/src/useSelectableItem.ts

```diff
@@ -30,7 +30,7 @@ export interface SelectableItemOptions {
   /**
    * Ref to the item.
    */
-  ref: RefObject<FocusableElement>,
+  ref: RefObject<FocusableElement | HTMLAnchorElement | null>,
```

Comment:
Instead of this, just remove `null` from the ref TS definition in useSideNavItem, I think that was a mistaken addition at some point and it's an unreleased component